src/manager/index.ts

```diff
@@ -67,4 +67,13 @@ export class Manager extends ApiConnection {
       this._requestFunc =
         requestFunc;
   }
+
+  public set onAuthFailed(callback: (url: string) => void) {
+    this.auth.onAuthFailed =
```

Comment:
I've never seen this syntax before to assign multiple variables the same value.. that's pretty neat!